apps/bfd-model/bfd-model-rda/src/main/java/gov/cms/bfd/model/rda/RdaClaimMessageMetaData.java

```diff
@@ -87,6 +87,22 @@ public class RdaClaimMessageMetaData {
   @Column(name = "received_date")
   private Instant receivedDate;
 
+  /** Identifies the particular phase associated with this claim (1, 2, 3) */
+  @Column(name = "phase")
+  private Short phase;
```

Comment:
@KyRobbins in the past we've used the primitive type here (https://github.com/CMSgov/beneficiary-fhir-data/pull/1191), will using the Short class cause any hibernate validation issues?

Reply:
For nullable columns we've used the boxed version and Hibernate is fine with that.

Reply:
I wouldn't expect it to since it's a nullable field, using wrapper classes is the only way to accurately contract with nullable fields.